servers/Fabric.Mcp.Server/src/Fabric.Mcp.Server.csproj

```diff
@@ -44,6 +43,33 @@
     <SelfContained>true</SelfContained>
     <IlcOptimizationPreference>Size</IlcOptimizationPreference>
     <IlcFoldIdenticalMethodBodies>true</IlcFoldIdenticalMethodBodies>
+    <CopyOutputSymbolsToPublishDirectory>false</CopyOutputSymbolsToPublishDirectory>
+  </PropertyGroup>
+  <!-- .NET Tool Package settings -->
+  <PropertyGroup Condition="'$(BuildNative)' == 'true'">
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <PackageType>McpServer</PackageType>
+    <PackageId>Microsoft.Fabric.Mcp</PackageId>
```

Comment:
I'd move the generic properties to Directory.Build.Props (maybe all conditional on BuildNative)
I'd move the server-specific ones like "PackageId" up to the property group with NpmPackageName

Reply:
Are these needed if we do packaging in a Pack-Dnx.ps1 script?

Reply:
Moving the properties into `Directory.Build.Props` is not quite feasible because of the MSBuild order. Some properties are intended to override previous properties but `Directory.Build.Props` is read first.